lib/alerts/check_timeline_manager.rb

```diff
@@ -11,7 +11,7 @@ def create_alerts
     # if course doesn't have any training module, it will not create an alert
     return if @course.training_module_ids.any?
     alert = Alert.create(type: 'CheckTimeline', course_id: @course.id,
-                         message: 'It appears you have removed either the timeline, week or block from your course.')
+                         message: 'It appears you have removed timeline from your course.')
```

Comment:
Per my earlier comment, I think this message should be reworded, as it's a message for admins rather than for the user who deleted the timeline. It's also not completely accurate, as deletion of the timeline is not the only condition that will trigger an alert.

I suggest moving the message itself into the alert subtype class file.

Reply:
So i don't need to add message while creating alert right?. As the message in the alert subtype class would be showed up. And what do you suggest as changes in message?

Reply:
Do i need to add message here? [](https://user-images.githubusercontent.com/72390769/213884087-6650b2ae-8e29-48af-8d95-2260fddae8c3.png)

Reply:
Adding the message when creating the alert is okay. I'm suggesting that the message string be defined in the class file, rather than here.